1684.count-the-number-of-consistent-strings.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/count-the-number-of-consistent-strings/
+ */
+
+/**
+ * @param {string} allowed
+ * @param {string[]} words
+ * @return {number}
+ */
+var countConsistentStrings = function(allowed, words) {
+    const allowedSet = new Set(allowed);
+    
+    //Number of consistent strings in the array
+    let result =　words.length;
+    
+    words.forEach((word) => {
+        for (let i = 0; i < word.length; i++) {
+            if (!allowedSet.has(word[i])) {
```

Comment:
```suggestion
            if (!allowed.includes(word[i])) {
```
You can change like this if you don't use Set for allowed.